test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
Even though the test doesn't validate it, it helps if the test data is **close** to what it actually is. Like when I started working on data-schema, I looked at the [test data](https://github.com/screwdriver-cd/data-schema/tree/master/test/data) to quickly pick up on what the data might look like. And it was really helpful that we had those very close to the real data even though it wasn't necessary for some fields (and I recalled Tiffany also commented on its usefulness). That was my reasoning to get into this discussion in the first place. 

Even though I still think `bitbucket.org:arnoldWeber/wyattLib:mybranch` is confusing to code readers since it looks more like `owner/repoName`, not `owner/slug`, it's over 1 line of code in test. So if you don't want to change it, I will concede.